src/Skoruba.IdentityServer4.STS.Identity/Controllers/AccountController.cs

```diff
@@ -129,10 +131,30 @@ public async Task<IActionResult> Login(LoginInputModel model, string button)
 
             if (ModelState.IsValid)
             {
-                var user = await _userResolver.GetUserAsync(model.Username);
+                TUser user;
+                //TODO: Code smell.  Evaulate if this huge method can be refactored.
+                if (_isMultiTenant)
```

Comment:
maybe it would be better to have one path? e.g all apps would be multi-tenant and have tenant by default or create two separate project templates (single and multiple tenants), otherwise, only one branch will be executed.